docs/Introduction.Android.md

```diff
@@ -145,6 +145,35 @@ Following device types could be used to control Android devices:
 `android.attached`. Connect to already-attached android device. The device should be listed in the output of `adb devices` command under provided `name`.
 Use this type to connect to Genymotion emulator.
 
+#### 5a. Using product flavors
+
+Instead of using `debug` and `release` configurations to Detox you may want to use your own [productFlavors](https://developer.android.com/studio/build/build-variants#product-flavors):
```

Comment:
Please rephrase as to explain that this is for projects that have custom product flavors, and that in _that_ case, the config should be applied a bit differently.